tools/make-config.py

```diff
@@ -642,7 +643,10 @@ def find_dups(threshold, dir1, dir2) -> dict[str, str]:
 
 
 def split(splat_config_path: str, disassemble_all: bool):
-    if disassemble_all:
+    with open(splat_config_path) as f:
+        c = yaml.load(f, Loader=yaml.SafeLoader)
+        config_disasm_all = c["options"]["disassemble_all"]
+    if disassemble_all and not config_disasm_all == True:
```

Comment:
This is a bit of a hack to workaround the splat bug, basically check if it's already set to true in the yaml and if so don't try to set the command argument. Doesn't seem to slow down make_config too much

The alternative would be what xee was testing which is making this option passed in via command line always in the Makefile